### PR TITLE
docs: add map usage

### PR DESCRIPTION
**Pull request by rfyiamcool**

### CODE REVIEW

File: example/main.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package main
```
Comment (caarlos0) [inline example/main.go:1]: probably better to update the current `Example`: https://github.com/caarlos0/env/blob/06b4d47785c9245e4424541ecb188bd426bda4dd/env_test.go#L1306

Comment (caarlos0) [reply example/main.go:1]: maybe would be good to put all the `Example.*` functions in a `examples_test.go` so its easier to find though

Comment (rfyiamcool) [reply example/main.go]: 😆  done

Comment (caarlos0) [reply example/main.go]: awesome, thank you!

**Pull request merged**